datafusion/functions/src/string/common.rs

```diff
@@ -35,19 +35,26 @@ use datafusion_expr::ColumnarValue;
 
 /// Append a new view to the views buffer with the given substr
 ///
-/// raw must be a valid view
-/// substr must be a valid substring of raw
+/// # Safety
+/// original_view must be a valid view
```

Comment:
what is valid view? 🤔

Reply:
Good call -- I tried to clarify in c8668df3b